<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инвентаризация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importWinBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>75</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Импорт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewdbBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>75</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотр БД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="print_int_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Печать
инвентарных
номеров</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="db_filename">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>inventarizaciya10.db</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="file_open_btn">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Используемая БАЗА ДАННЫХ</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
